developer-docs-site/sidebars.js

```diff
@@ -222,25 +222,83 @@ const sidebars = {
         "standards/wallets",
       ],
     },
+    {
+      type: "category",
+      label: "Aptos APIs",
+      link: { type: "doc", id: "apis/index" },
+      collapsible: true,
+      collapsed: true,
+      items: [
+        "apis/fullnode-rest-api",
+        "apis/aptos-labs-developer-portal",
+        {
+          type: "category",
+          label: "Indexing",
+          collapsible: true,
+          collapsed: true,
+          link: { type: "doc", id: "indexer/indexer-landing" },
+          items: [
+            {
+              type: "category",
+              label: "Indexer API",
+              link: { type: "doc", id: "indexer/api/index" },
+              collapsible: true,
+              collapsed: true,
```

Comment:
Please make backlinks from old URLs to the new ones

Reply:
actually the URLs for the indexer docs haven't changed. This is just a change in where they appear in the sidebar structure.